src/client/gui/lib/vm_details/vm_details_general.dart

```diff
@@ -129,6 +130,7 @@ class VmStat extends StatelessWidget {
 
 class GeneralDetails extends ConsumerWidget {
   final String name;
+  static const double preferredHeight = 50;
```

Comment:
Perhaps this can benefit from a better name. Something like `defaultHeightForEntry` to make it clear which height it is referencing. Also, specifying the type is not really needed if it can be inferred, so you could do something like `static const variable = 50.0;`

Reply:
Based on our discussion elsewhere, I updated to baseVmStatsHeight. I ended up not modifying the name of the parameter itself since, as you say, the better approach would be for the widget to just derive the height from the child. Let me know if this is what you had in mind.